lib/common-args.js

```diff
@@ -36,6 +36,11 @@ module.exports = function commonArgs (app) {
     type: 'string',
     description: 'Path to prepend to $PATH when running tests'
   })
+  .option('append', {
+    alias: 'a',
+    type: 'boolean',
+    description: 'Turns on append to file mode'
```

Comment:
I'd like a slightly more in depth description here. Mentioning that it will append rather than replace.